pipelines/packageupmpublic.yaml

```diff
@@ -0,0 +1,14 @@
+# Create UPM packages for public OFFICIAL release
+
+variables:
+- template: config/settings.yml
+
+jobs:
+- job: CreatePublicUPMArtifacts

```

Comment:
For future reference, it would be good to file an issue to move this to a multi-stage pipeline, rather than two separate jobs that run.

Generally speaking when releasing we should kick off a single pipeline which generates a preview/canary -> verify that -> deploy to prod. It would be good just to record this for historical purposes but actually doesn't have much of an impact on the overall given the rest of the infra we have right now.